front/entityconfig.form.php

```diff
@@ -40,6 +40,10 @@
 
 if (isset($_POST['update'])) {
    $entityConfig = new PluginFormcreatorEntityconfig();
-   $entityConfig->update($_POST);
+   if ($entityConfig->getFromDBByCrit(['entities_id' => (int) $_POST['entities_id']])) {
+      $_POST['id'] = $entityConfig->getID();
+      unset($_POST['entities_id']);
+      $entityConfig->update($_POST);
+   }
```

Comment:
The table glpi_plugin_formcreator_entityconfigs extends the table glpi_entities to store parameters.

Until this PR the column ID of glpi_plugin_formcreator_entityconfigs must match the ID og the entity. I removed this behabiour by adding a foreign key to the table. I believe this is better.